src/content/documentation/develop/manifest-v3-migration-guide.md

```diff
@@ -283,13 +283,14 @@ In Manifest v2, Firefox extensions support the use of the `chrome.*` namespace w
 ## Migration checklist
 
 - Update the manifest.json key `manifest_version` to `3`.
-- If your extension adds a search engine, add a local icon and reference it in the  manifest.json key `chrome_settings_overrides.search_provider.favicon_url`.
+- If your extension adds a search engine, add a local icon and reference it in the manifest.json key `chrome_settings_overrides.search_provider.favicon_url`.
 - Remove any host permissions from the manifest.json keys permissions and `optional_permissions` and add them to the `host_permissions` key.
 - Rename the manifest.json key `browser_action` to `action` and update any API references from `browser.browserAction` to `browser.action`.
 - Convert background pages to be non-persistent.
 - Move the extension’s CSP to the the manifest.json key ‘content_security_policy.extension_pages’ and update the CSP to conform to Manifest V3 requirements.
 - Move any arbitrary strings executed as scripts to files and update your code to use the Scripting API.
 - The add-on ID is required to publish your extension. Make sure to add one in the `browser_specific_settings.gecko.id` manifest.json key.
+- The format of the `version` manifest.json key has evolved and became simpler: the `version` must be a string with 1 to 4 numbers separated with dots (e.g. `1.2.3.4`). Each number can have up to 9 digits and leading zeros are not allowed (e.g. `2.01` is forbidden but `2.1` is allowed).
```

Comment:
Leading zeroes were already invalid before in the linter, so it's unlikely to help anyone.

I suggest to simplify the description to only what's been changed and remove the "and leading... allowed)". To highlight the change, put "Letters are no longer allowed." after "simpler:". Then readers won't have to wonder what changed and what they have to do 

MDN should list the constraints in the documentation, and the EW can link to that article.

Reply:
> Leading zeroes were already invalid before in the linter, so it's unlikely to help anyone.

`1.01a` was accepted by the linter.

Reply:
But there is a letter in there which triggered the special case.

The "leading zero" remark could raise questions about the validity of a plain zero.

I'm trying to find a balance between what people have to read and what actionable parts there are. The sniff test "does version contain a letter" covers most if not all cases. For a precise description a reference to specific description seems appropriate and sufficient.

Something like this would work:

> "The `version` key in manifest.json has a stricter format. Letters are no longer allowed, only up to four numbers separated with a dot. For the precise description, see the [version documentation on MDN]"

Reply:
I made some edits. That should be good until we have the page on MDN updated (without that being a blocker).